stdlib/array.pyi

```diff
@@ -21,15 +21,19 @@ class array(MutableSequence[_T], Generic[_T]):
     @property
     def itemsize(self) -> int: ...
     @overload
-    def __init__(self: array[int], __typecode: _IntTypeCode, __initializer: bytes | Iterable[int] = ...) -> None: ...
+    def __init__(self: array[int], __typecode: _IntTypeCode, __initializer: bytes | bytearray | Iterable[int] = ...) -> None: ...
```

Comment:
bytes and bytearray specifically are treated differently. Arbitrary non-iterable buffers (e.g., `pickle.PickleBuffer`) don't work.